tensorbay/cli/tag.py

```diff
@@ -24,7 +24,16 @@ def _implement_tag(obj: Dict[str, str], tbrn: str) -> None:
     gas = get_gas(**obj)
     dataset_client = get_dataset_client(gas, info)
 
-    _list_tags(dataset_client)
+    if tag_name:
+        _create_tag(dataset_client, info, tag_name)
+    else:
+        _list_tags(dataset_client)
+
+
+def _create_tag(dataset_client: DatasetClientType, info: TBRN, tag_name: str) -> None:
+    dataset_client.create_tag(name=tag_name)
+    tag_tbrn = TBRN(info.dataset_name, revision=dataset_client.status.commit_id).get_tbrn()
+    click.echo(f"{tag_tbrn} is created successfully")
```

Comment:
About the message of this command, here is an example according this code logic:
```console
$ gas tag tb:KITTI v1.1.1
tb:KITTI@e7ca2b48-9ae6-43ce-a945-efeb11cfaae9 is created successfully
```
Do you think is reasonable to print commit_id?